jdbc persistence: tableNamePrefix error on startup

Hi @lewie ,

I am using the OH2 build number 433 from Jenkins with your JDBC persistence included in it. On startup, I get (amongst other debug traces) the following error:

2015-12-04 14:51:52 [ERROR] [zaxxer.hikari.util.PropertyElf:140  ] - Property tableNamePrefix does not exist on target class com.zaxxer.hikari.HikariConfig

my conf/services/jdbc.cfg is:

```
#
# JDBC Binding settings
#
tableNamePrefix=Item
tableUseRealItemNames=false
tableIdDigitCount=4

# connection string url
url=jdbc:postgresql://my-server:5432/openhab

# driver class name
driver=org.postgresql.Driver

# username
user=secret

# password
password=secret,too

sqltype.string=VARCHAR(255)
```

Do you have any idea?

--- a/bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/internal/JdbcConfiguration.java
+++ b/bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/internal/JdbcConfiguration.java
@@ -166,7 +166,8 @@ public class JdbcConfiguration {
 
         String np = (String) configuration.get("tableNamePrefix");
         if (StringUtils.isNotBlank(np)) {
-            dBDAO.databaseProps.setProperty("tableNamePrefix", np);
+	    tableNamePrefix = np;
+	    logger.debug("JDBC::updateConfig: tableNamePrefix={}", tableNamePrefix);
         }
 
         String dd = (String) configuration.get("numberDecimalcount");

